tools/dhtnode.cpp

```diff
@@ -55,16 +55,23 @@ void print_help() {
               << "  ll         Print basic information and stats about the current node." << std::endl
               << "  ls [key]   Print basic information about current search(es)." << std::endl
               << "  ld [key]   Print basic information about currenty stored values on this node (or key)." << std::endl
-              << "  lr         Print the full current routing table of this node" << std::endl;
+              << "  lr         Print the full current routing table of this node." << std::endl;
+
+#if OPENDHT_PROXY_SERVER
+    std::cout << std::endl << "Operations with the proxy:" << std::endl
```

Comment:
add a command line argument to start a server with dhtnode.

For instance:
`dhtnode --proxyserver 8080 -b bootstrap.ring.cx`
to start a server on port 8080